src/sphinx.md

```diff
@@ -96,6 +102,7 @@ transformed.
 
 The artificial delay should be calculated as `exp_random(delay_seed) * mean_delay`, where
 `mean_delay` is the mean forwarding delay (see the [parameters chapter](./parameters.md)).
+// TODO state if something prevent not following this rule.
```

Comment:
Not sure what you mean -- it should always be possible to do this?

Reply:
actually "should" means that nothing enforce this.
maybe we can use uper case SHOULD that indicate expected behavior that implementors should follow but that nothing enforce.
oposed to a MUST that should make the node appears like a bad actor.

Reply:
I've used "may" for optional behaviour. I'm not particularly convinced that distinguishing between must and should is that useful. At some point there will hopefully be an incentivisation scheme, at which point such a distinction might make sense.

Reply:
Putting all MAY SHOULD MUST uppercase could make change of behavior (eg when incentivisation required or defined) easier and it is pretty common in specs I think.